changes/2020-07-15_specify-behavior-for-raw-aes-keyring-for-invalid-EC/change.md

```diff
@@ -0,0 +1,99 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Specify behavior for RawAESKeyring for invalid input EC
+
+## Affected Features
+
+This serves as a reference of all features that this change affects.
+
+| Feature                                               |
+| ----------------------------------------------------- |
+| [raw-aes-keyring](../../framework/raw-aes-keyring.md) |
+
+## Affected Specifications
+
+This serves as a reference of all specification documents that this change affects.
+
+| Specification                                         |
+| ----------------------------------------------------- |
+| [raw-aes-keyring](../../framework/raw-aes-keyring.md) |
```

Comment:
Should these be SHA's, so we know the version of the spec that this changed?

Reply:
I cribbed the format from other open PRs, which don't seem to do this. Your suggestion makes sense though, seems like we need to better standardize the change doc template.